unfoldingWord/translationCore
# removed unnecessary resources saving functions from codebase

#### This pull request addresses:
- Fixed bug with verse Edits.
- Removed unnecessary resources saving functions from codebase


#### How to test this pull request:
(test with both tN & tW)
- Make a verse edit.
- Load a fresh project and test that all Bible verses render in the scripture pane and verse Check components.

- Reload the same project and test the same thing. 

---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1878)
